src/prof_sys.c

```diff
@@ -97,13 +97,17 @@ prof_backtrace_impl(void **vec, unsigned *len, unsigned max_len) {
 
 	cassert(config_prof);
 	assert(vec != NULL);
-	assert(max_len == PROF_BT_MAX);
+	assert(max_len <= PROF_BT_MAX);
 
 	_Unwind_Backtrace(prof_unwind_callback, &data);
 }
 #elif (defined(JEMALLOC_PROF_GCC))
 static void
 prof_backtrace_impl(void **vec, unsigned *len, unsigned max_len) {
+/* According to the official document, __builtin_return_address
+ * takes in a constant, instead of a variable.
+ * Thus, we use the macro below.
+*/
```

Comment:
The comments format look off. You will want to configure your editor to do it for you, i.e. 80-character max line, and comment format should be:

```
/* This is a single line comment. */

/*
 * This is a multi-line comment.  Follow sentence-ending periods with two
 * spaces.  Consider that we display code with fixed-width fonts; embrace the
 * old 20th-century convention and like it.
 */
```
From this page: https://github.com/jemalloc/jemalloc/wiki/Coding-Style

Reply:
Modified.